### ISSUE TITLE
Power saving mode

### ISSUE THREAD

**Issue by varomix** [Yatekii/imgui-wgpu-rs#29]
Hi
I want to make a desktop app that doesn't need to run 60 fps all the time
could something like this be implemented fo wgpu here? 

https://github.com/ocornut/imgui/pull/2749

thank you

**Comment by cwfitzgerald** [comment]
What would be required of us to do this? We don't handle any of the interaction with the system, all we do is render when we are asked to. This seems like it's on the application to deal with and only call us when a redraw is needed.

**Comment by Yatekii** [comment]
Yeah, this id definitely something the actual application needs to handle. the framerate is not handled by imgui-wgpu :)
